### PR TITLE
Add placeholder to SearchBar's schema

### PR DESCRIPTION
**Pull request by Klynger**
#### What problem is this solving?

As the title says.

#### How should this be manually tested?

[Workspace](https://searchprop--storecomponents.myvtex.com/admin/cms/site-editor)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [X] Updated `CHANGELOG.md`.
- [ ] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: react/components/SearchBar/index.js [hunk 1]
```diff
@@ -88,6 +88,17 @@ SearchBarContainer.contextTypes = {
   navigate: PropTypes.func,
 }
 
+SearchBarContainer.schema = {
+  title: 'Search Bar',
+  type: 'object',
+  properties: {
+    placeholder: {
+      type: 'string',
+      name: 'place',
```
Comment (klzns) [inline react/components/SearchBar/index.js]: Add translation

File: react/components/SearchBar/index.js [hunk 2]
```diff
@@ -88,6 +88,17 @@ SearchBarContainer.contextTypes = {
   navigate: PropTypes.func,
 }
 
+SearchBarContainer.schema = {
+  title: 'Search Bar',
```
Comment (klzns) [inline react/components/SearchBar/index.js]: Add translation

File: messages/en.json [hunk 3]
```diff
@@ -167,5 +167,7 @@
   "admin/editor.skuSelector.title": "SKU Selector",
   "admin/editor.skuSelector.description": "Component that displays different given variations of an item for user to choose",
   "admin/editor.skuSelector.seeMoreLabel.title": "See more label",
-  "admin/editor.skuSelector.seeMoreLabel.description": "Label of see more button. The string must have a {quantity} placeholder to show the appropriate remaining items available. Example: \"See {quantity} more\""
+  "admin/editor.skuSelector.seeMoreLabel.description": "Label of see more button. The string must have a {quantity} placeholder to show the appropriate remaining items available. Example: \"See {quantity} more\"",
+  "admin/editor.search-bar.title": "Search bar",
+  "admin/editor.search-bar.placeholder.title": "Place holder"
```
Comment (klzns) [inline messages/en.json]:
```suggestion
  "admin/editor.search-bar.placeholder.title": "Placeholder"
```

File: messages/context.json [hunk 4]
```diff
@@ -167,5 +167,7 @@
   "admin/editor.skuSelector.title": "Title of SKU Selector component",
   "admin/editor.skuSelector.description": "Description of SKU Selector component",
   "admin/editor.skuSelector.seeMoreLabel.title": "Title of seeMoreLabel property",
-  "admin/editor.skuSelector.seeMoreLabel.description": "Description of seeMoreLabel property"
+  "admin/editor.skuSelector.seeMoreLabel.description": "Description of seeMoreLabel property",
+  "admin/editor.search-bar.title": "Search bar",
+  "admin/editor.search-bar.placeholder.title": "Place holder"
```
Comment (Klynger) [inline messages/context.json]:
```suggestion
  "admin/editor.search-bar.placeholder.title": "Placeholder"
```

**Pull request merged**